packages/settings-library/src/settings_library/celery.py

```diff
@@ -35,6 +35,13 @@ class CelerySettings(BaseCustomSettings):
         ),
     ] = True
 
+    CELERY_POOL: Annotated[
+        Literal["prefork", "eventlet", "gevent", "solo", "threads"],
+        Field(
+            description="Type of pool to use. One of: prefork, eventlet, gevent, solo, threads. See https://docs.celeryq.dev/en/stable/userguide/workers.html#choosing-a-concurrency-implementation for details.",
```

Comment:
Can you doc please what you state in the PR description i.e. for cpu-bound tasks use 'prefork' otheriwse ...etc  (the fragment in the link is not pointing to any section!)